tests/legacy-cli/e2e/tests/commands/add/npm-env-vars.ts

```diff
@@ -0,0 +1,35 @@
+import { expectFileNotToExist, expectFileToExist, writeFile } from '../../../utils/fs';
+import { getActivePackageManager, setRegistry } from '../../../utils/packages';
+import { git, ng } from '../../../utils/process';
+import {
+  createNpmConfigForAuthentication,
+  resetNpmEnvVars,
+  setNpmEnvVarsForAuthentication,
+} from '../../../utils/registry';
+
+export default async function () {
+  const packageManager = getActivePackageManager();
+
+  if (packageManager === 'npm') {
+    try {
+      const command = ['add', '@angular/pwa', '--skip-confirmation'];
+
+      // Environment variables only
+      await expectFileNotToExist('src/manifest.webmanifest');
+      await writeFile('.npmrc', '');
```

Comment:
Doesn't seem to be needed.

```suggestion
```

Reply:
I wanted to try and simulate the specific use case where an *.npmrc* did not exist.  Now that I understand how these tests work a little more, I can see why this step probably isn't needed.  I'll take it out.